htsohm/simulation/gas_adsorption.py

```diff
@@ -0,0 +1,216 @@
+import sys
+import os
+import subprocess
+import shutil
+from datetime import datetime
+from uuid import uuid4
+
+import htsohm
+from htsohm import config
+from htsohm.material_files import write_cif_file, write_mixing_rules
+from htsohm.material_files import write_pseudo_atoms, write_force_field
+from htsohm.simulation.calculate_bin import calc_bin
+
+def write_raspa_file(filename, material):
+    """Writes RASPA input file for simulating gas adsorption.
+
+    Args:
+        filename (str): path to input file.
+        material (Material): material record.
+
+    Writes RASPA input-file.
+
+    """
+    simulation_cycles      = config['simulations']['gas_adsorption']['simulation_cycles']
+    initialization_cycles  = config['simulations']['gas_adsorption']['initialization_cycles']
+    external_temperature   = config['simulations']['gas_adsorption']['external_temperature']
+    external_pressure      = config['simulations']['gas_adsorption']['external_pressure']
+    adsorbate              = config['simulations']['gas_adsorption']['adsorbate']
```

Comment:
if we passed the simulation config, these would just look like:

simulation_cycles = config['simulation_cycles']